x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/anomaly_detection_flyout.tsx

```diff
@@ -17,8 +17,8 @@ import { MetricK8sModuleProvider } from '../../../containers/ml/modules/metrics_
 import { useActiveKibanaSpace } from '../../../hooks/use_kibana_space';
 
 export const AnomalyDetectionFlyout = ({
-  hideJobType,
-  hideSelectGroup,
+  hideJobType = false,
```

Comment:
Why do we need those default values?

Reply:
It's better than letting them be `undefined` in case it's not informed

Reply:
I asked because we have them as optional also in other places where we check the value but it's Ok to have them set to false by default.